src/mlpack/methods/decision_tree/all_categorical_split.hpp

```diff
@@ -103,7 +103,8 @@ class AllCategoricalSplit
       const size_t minimumLeafSize,
       const double minimumGainSplit,
       double& splitInfo,
-      AuxiliarySplitInfo& aux);
+      AuxiliarySplitInfo& aux,
+      FitnessFunction& fitnessFunction);
```

Comment:
Can you add documentation for the `fitnessFunction` parameter to the function documentation?  (This applies for the other functions where you added it too. :+1:)